.github/workflows/checks.yml

```diff
@@ -87,11 +87,20 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-project
       - name: Create requirements.txt
-        run: poetry export --without-hashes --format=requirements.txt > requirements.txt
+        run: poetry export --output requirements_tmp.txt --without-hashes
+      - name: Filter requirements.txt
+        run: grep -v "oscrypto@ git" requirements_tmp.txt > requirements.txt
+      - name: Verify requirements.txt
+        run: ls -l requirements.txt
+      - name: Print requirements.txt
+        run: |
+          echo "Contents of requirements.txt:"
+          cat requirements.txt
       - uses: pypa/gh-action-pip-audit@v1.0.8
         with:
           inputs: requirements.txt
           ignore-vulns: |
+            PYSEC-2022-43162
```

Comment:
This line can be taken out now - might just need to sync with `main` for this too!